.dockerignore

```diff
@@ -1,22 +1,13 @@
-alice
-bob
-.dockerignore
-node_modules
-scripts
-test
-zero
-*.db
-charts
-helmfile.d
-build
-.circleci
-.editorconfig
-.git
-.github
-.gitignore
-.idea
-combined.log
-docker-compose.yml
-node_modules/
-
-*/**/node_modules/
\ No newline at end of file
+*
+!apps
+!candidates
```

Comment:
candidates are not necessary in the docker image, they are http fetched or passed via config file